src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -224,6 +224,19 @@ public Position getPosition() {
         });
   }
 
+  @GraphQLQuery(name = "allPreviousPositions")
```

Comment:
as this changes the GraphQL schema, you need to state that in the PR body
- [x] graphql schema has changed

Reply:
I've updated the PR

Reply:
disclaimer: I have not looked into the semantics of the bug and the specific fix...
Do we need to change the schema to fix this bug? In the changed schema now we have:
`previousPositions`
`allPreviousPositions`
It is not very clear why we have 2 similarly named fields. If the functionality of `allPreviousPositions` is required and `previousPositions` is not enough, can the two be combined in one?

Reply:
GraphQLQuery and  GraphQLInput removed. 
Graphql schema change cancelled.

Reply:
GraphQLQuery and GraphQLInput removed.
Graphql schema change cancelled.